nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -169,6 +143,13 @@
         .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
         .build();
 
+    protected static final PropertyDescriptor IGNORE_NULL = new Builder()
+        .name("Ignore NULL")
+        .description("Ignore NULL on Kudu Put Operation, Update only non-Null columns if set true")
+        .defaultValue("false")
```

Comment:
Before we added this property to the processor, did he default to "false"? just want to be sure we are backward compatible here.

Reply:
@pvillard31 this is the first time we are adding this IGNORE_NULL field in this processor

Reply:
Yes right and I just noticed Joe did the same comment before, so we are good.